src/utils/gtag.js

```diff
@@ -1,7 +1,10 @@
 window.dataLayer = window['dataLayer'] || []
+
+
+/** Create google analytics tag. */
 export default function gtag() {
   const dataLayer = window.dataLayer
-  dataLayer.push(arguments)
```

Comment:
I think this is valid per:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments
.. maybe add this link as a comment and then eslint disble next line

Reply:
not sure about indentation